resource/ui/ts/models/proto.ts

```diff
@@ -243,9 +243,46 @@ module Models {
          * Source message = "TimeSeriesQueryRequest"
          */
         export interface QueryRequestSet {
-            start_nanos:number;
-            end_nanos:number;
-            queries:QueryRequest[];
+            start_nanos: number;
+            end_nanos: number;
+            queries: QueryRequest[];
+        }
+
+
+        /*****************************
+         * /proto/log.proto
+         ****************************/
+
+        /**
+         * Arg represents an argument passed to a log entry.
+         *
+         * Source message = "LogEntry.Arg"
+         */
+        export interface Arg {
+            type: string;
+            str: string;
+            json: string;
+        }
+
+        /**
+         * LogEntry represents a cockroach structured log entry.
+         *
+         * Source message = "LogEntry"
+         */
+        export interface LogEntry {
+            severity: number;
```

Comment:
Yes, but I don't know if this is the correct place for it.  I think this file should be purely typings direct from the cockroach protos only.